RTProtocol.cs

```diff
@@ -24,6 +24,14 @@ public enum StreamRate
         RateFrequencyDivisor
     }
 
+    public enum ReceiveResponseType
+    {
+        success,
+        timeout,
+        error,
+        disconnect
+    };
```

Comment:
This is a redeclaration of the other (nested) enum `Response::ResponseType`. Exactly the same enumeration names but different order for last two... Should share the same enum declaration to avoid confusion.

Reply:
Only use the enum in RTNetwork